packages/store/internals/src/memoize.ts

```diff
@@ -59,10 +43,5 @@ export function ɵmemoize<T extends (...args: any[]) => any>(
     lastArgs = arguments;
     return lastResult;
   }
-  (<any>memoized).reset = function () {
-    // The hidden (for now) ability to reset the memoization
-    lastArgs = null;
-    lastResult = null;
-  };
```

Comment:
The removal of this function is unrelated to this PR.
And besides, what is the motivation to remove it?

PS. The PR also requires an update to the changelog

Reply:
I can remove it in another PR, but that part should not be included in the code unless it’s ever used. Extra work is bad, so unless there’s a valid argument for including that extra code, it shouldn’t be there.